public/css/widget/host-meta-info.less

```diff
@@ -64,7 +64,11 @@
 }
 
 .controls > .host-meta-info {
-  margin-top: -.5em;
+  padding: .5em 0;
+}
+
+.controls > #host-meta-info.collapsed {
```

Comment:
There is already a suitable rule above, why not use it? (Instead of relying on an id which isn't for CSS rules (if there's a class) to begin with.)

Reply:
What do you think of 186fd62a.

Also associated af4cf52c.

(Please take `@ToDo` comments into account)